BuggyProject/ViewController.swift

```diff
@@ -18,16 +18,27 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
         validImageView?.backgroundColor = UIColor.red
         NotificationCenter.default.addObserver(self, selector: #selector(ViewController.textFieldDidChange(_:)), name: NSNotification.Name.UITextFieldTextDidChange , object: self.textField);
     }
 
     // MARK: Actions
     @IBAction func bTap() {
+        searches.append((self.textField?.text)!)
```

Comment:
can we unwrap this optional safely? `!` should be avoided unless you have a unit test that can demonstrated that the value will never be `nil`